src/main/java/uk/gov/digital/ho/hocs/queue/migration/MigrationMessageHandler.java

```diff
@@ -14,17 +14,21 @@ public class MigrationMessageHandler extends BaseMessageHandler {
 
     private final MigrationCaseService migrationCaseService;
 
+    private final MigrationValidator migrationValidator;
     public MigrationMessageHandler(
             @Value("${message.ignored-types}") List<String> ignoredMessageTypes,
-            MigrationCaseService migrationCaseService
+            MigrationCaseService migrationCaseService,
+            MigrationValidator migrationValidator
     ) {
         super(ignoredMessageTypes);
         this.migrationCaseService = migrationCaseService;
+        this.migrationValidator = migrationValidator;
     }
 
     @Override
     public void handleMessage(String message, String messageId) throws Exception {
-        log.info("Received new message MessageId : {}, {}", messageId, message);
```

Comment:
This log line would still be useful to have I think - even if we strip back to just the messageId. I would expect the migrator service to log out the mesageId too and that way we can correlate for any issues.